src/Sylius/Bundle/AddressingBundle/Factory/ZoneFactory.php

```diff
@@ -62,11 +62,11 @@ public function createTyped($type)
     /**
      * {@inheritdoc}
      */
-    public function createWithMembers(array $members)
+    public function createWithMembers(array $membersCodes)
```

Comment:
It suggests that there is an array of codes belonging to one member, `$membersCodes` is all right.